pkg/manager-server/file_system_lvm.go

```diff
@@ -171,7 +172,24 @@ func (f *FileSystemLvm) Create(devices []string, opts FileSystemOptions) error {
 		return err
 	}
 
-	return nil
+	// Ensure that the Logical Volume exists prior returning; otherwise a future 'mkfs' command may fail.
+	/*
+		NJT: Can't do this yet since to Persistent() controller all errors are fatal and perform a Rollback.
+		if _, err := os.Stat(f.devPath()); os.IsNotExist(err) {
+			return ec.NewErrorNotReady().WithCause(fmt.Sprintf("logical volume '%s' does not exist", f.devPath()))
+		}
+	*/
+
+	// HACK: Wait up to 10 seconds for the path to come ready
+	for i := 0; i < 10; i++ {
```

Comment:
Throw the 10 in a constant even though it's a hack

Reply:
Can you explain your reasoning here?

Reply:
Magic number + it would be easier to change at the top of the file if it needed to decrease/increase. But also not a big deal.

Reply:
Exported constants belong at the top of a file; But the preference should be toward locality otherwise.

In case the comment wasn't clear, I re-wrote it so it's more obvious what the code is doing.